web/src/containers/admin/ManageUserTable.test.js

```diff
@@ -48,7 +48,10 @@ describe('<ManageUserTable />', () => {
   test('shows loading when data is fetching', () => {
     props = {
       tab: 'active',
-      isFetching: true
```

Comment:
This seems like another case for the <Foo {...defaultProps} { ...props} /> pattern and maybe a table test.  I don't want to hold this up though.  Do we do Tech debt tickets?